docker/README.md

```diff
@@ -0,0 +1,30 @@
+You can use Docker to easily try out or test sendgrid-csharp.
+
+# Quickstart
+
+1. Install Docker on your machine.
+2. Run `docker run -it sendgrid/sendgrid-csharp`.
+
+# Info
+
+This Docker image contains
+ - `sendgrid-csharp`
```

Comment:
Here's what I tried:
```bash
$ git pull https://github.com/FireEater64/sendgrid-csharp.git initial-dockerfile
$ git remote add upstream https://github.com/sendgrid/sendgrid-csharp.git
$ git merge upstream master
$ git fetch
$ docker run -it -v /Users/mbernier/Sites/branches/FireEater64:/mnt/sendgrid-csharp sendgrid/sendgrid-csharp
```
The error I got: 
```
Unable to find image 'sendgrid/sendgrid-csharp:latest' locally
docker: Error response from daemon: pull access denied for sendgrid/sendgrid-csharp, repository does not exist or may require 'docker login'.
```
I also tried:
```bash
$ docker run -it sendgrid/sendgrid-csharp
```
and got a similar error:
```
Unable to find image 'sendgrid/sendgrid-csharp:latest' locally
docker: Error response from daemon: pull access denied for sendgrid/sendgrid-csharp, repository does not exist or may require 'docker login'
```

Do you know what I did incorrectly?

Reply:
@FireEater64 Can you please give this a look?

Reply:
Hey @mbernier - I wrote these instructions under the assumption that someone @sendgrid would build and push this container to the docker hub. Since I'm not a member of the [sendgrid org](https://hub.docker.com/u/sendgrid/), I can't do it myself, at least not under that name.

I can rework the instructions if you'd like - but to build the container yourself, just check out the branch and run `docker build -t sendgrid/sendgrid-csharp .` Then the instructions you mentioned above should work 😄

Reply:
Ahh, gotcha! Yes, we will eventually get this into docker hub. Until then, we want to make sure that people can run it from the library.

When we add it to docker hub we will update the docs accordingly!! 
Can you please make the suggested change? Then we will get this merged. 

Thank you!

Reply:
Of course - should be done :+1: